include/z64.h

```diff
@@ -615,227 +624,257 @@ typedef struct {
 
 typedef void(*fault_update_input_func)(Input* input);
 
+typedef struct {
+    /* 0x00 */ u8 unk_00;
+    /* 0x01 */ u8 unk_01;
+    /* 0x02 */ u8 unk_02;
+} UnkAudioStruct;
```

Comment:
This structure is the `OcarinaStaff` struct from [Krimtonz' work on OoT audio](https://github.com/zeldaret/oot/blob/7849742e36410d54c4ce97556fa2e47324a276f1/include/z64audio.h#L1178), if you want to add it.